src/odemis/gui/cont/views.py

```diff
@@ -325,31 +407,48 @@ def _on_chamber_state(self, state):
             # sample have probably nothing in common with this new sample
             self._data_model.stage_history.value = self._data_model.stage_history.value[-1:]
 
+    def _update_ovv(self):
+        """ Update the overview image with the currently active stream. """
+        if self._data_model.streams.value and self.curr_s \
```

Comment:
Is it useful to check `self._data_model.streams.value` is not empty? If it's empty, `curr_s` should be `None`. Or are there special cases? In that case add a comment.